.github/workflows/release_production.yml

```diff
@@ -42,6 +42,18 @@ jobs:
     - name: Extract version
       id: extract_version
       run: echo "RELEASE_VERSION=${version}" >> $GITHUB_OUTPUT
+    - name: Build Docker image and push to Docker hub and K8S
```

Comment:
Can we move this after cf pages deployment since cf pages is our primary deployment target?

Reply:
you are right, yess let me do it